lib/v2/6vdy/radar.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+    'hao6v.cc': {
+        _name: '6v电影',
+        '.': [
+            {
+                title: '最新电影',
+                docs: 'https://docs.rsshub.app/multimedia.html#_6v-dian-ying',
+                source: ['/', '/gvod/zx.html'],
+                target: '/6vdy/latestMovies',
+            },
+            {
+                title: '最新电视剧',
+                docs: 'https://docs.rsshub.app/multimedia.html#_6v-dian-ying',
+                source: ['/', '/gvod/dsj.html'],
+                target: '/6vdy/latestTVSeries',
+            },
+        ],
```

Comment:
You can create a variable for these entries instead of copy-pasting them 3 three times